### ISSUE TITLE
Preview: Not showing math for indented equation/split environment

### ISSUE THREAD

**Issue by sballin** [SublimeText/LaTeXTools#936]
Preview shows up correctly for 

    \begin{equation}
    \begin{split}
    ...
    \end{split}
    \end{equation}

but does not show up when the whole block or just the `\end{equation}` line is indented by any number of spaces.

Also, preview doesn't show up for

    \[
    \begin{split}
    ...
    \end{split}
    \]

which should be valid syntax.

**Comment by r-stein** [comment]
I assume

``` tex
\(
\begin{split}
...
\end{split}
\)
```

is not valid. The preview is currently wrapped into the environment append with * or into `\(...\)`, which I should change.

**Comment by Joshua1989** [comment]
Yes, I met similar problem when I use IEEEeqnarray (I add this environment into LaTeX.sublime-syntax).
An IEEEeqnarray block will look like below
```
\begin{IEEEeqnarray*}{rCl}
            x &=& 1 \\
            y &=& 2 \\
\end{IEEEeqnarray*}
```

This works fine if there is no indentation in the last line `\end{IEEEeqnarray*}`.
However, when the last line `\end{IEEEeqnarray*}` is indented, the math live actually converts it into
```
\(
{rCl}
            x &=& 1 \\
            y &=& 2 \\
\)
```
which gives extra `{rCl}` at front of equation and does not give correct aligning.

I also implement the tikz live preview naively by simply setting tikz environment as math environment, it is not perfect but meets my basic demand.
Similarly, if the last line `\end{tikzpicture*}` is indented, the content will be wrapped by `\(   \)` and fails to create tikz preview.

Well for tikz it is a long term issue, but I think it is worthy to modify the environment identification strategy, especially for this indentation problem.

**Comment by r-stein** [comment]
Seems I have used the wrong region ("full last line" instead of "last line behind scope end"). This should also work after the next release.

**Comment by r-stein** [comment]
Fixed in `v3.12.8`
